notebook.rst

```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The Jupyter Notebook (formerly known as IPython notebook)
 ======================
```

Comment:
Seems to render for me when I build it, though it is quite long:

![image](https://cloud.githubusercontent.com/assets/83444/14222859/f8826b82-f82a-11e5-81b2-fb676e7b9890.png)